mne_bids/utils.py

```diff
@@ -359,7 +359,7 @@ def _estimate_line_freq(raw, verbose=False):
     if len(picks) < 5:
         warn("Estimation of line frequency only "
              "supports 'meg', 'eeg', 'ecog', or 'seeg'.")
-        return None
+        return -1
```

Comment:
-1 ? are you a C programmer now?

Reply:
I moved it to where it was written to the file, I think that's better. Good one ;)